Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "86c37a92-59a7-444b-93c7-220fcf91e29c",
+    "name": "JenkinsDownloadArtifacts",
+    "friendlyName": "Jenkins Download Artifacts",
+    "description": "Download artifacts produced by a Jenkins job",
+    "helpMarkDown": "Download artifacts produced by a [Jenkins](https://jenkins.io/) job.",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft",
+    "demands": [],
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded": false
+        } 
+    ],
+    "instanceNameFormat": "Download artifacts produced by $(jobName)",
+    "inputs": [
+      {
+        "name": "serverEndpoint",
+        "type": "connectedService:Jenkins",
+        "label": "Jenkins service endpoint",
+        "defaultValue": "",
+        "required": true,
+        "helpMarkDown": "Select the service endpoint for your Jenkins instance.  To create one, click the Manage link and create a new Jenkins Service Endpoint."
+      },     
+      {
+        "name": "jobName",
+        "type": "string",
+        "label": "Job name",
+        "defaultValue": "",
+        "required": true,
+        "helpMarkDown": "The name of the Jenkins job to download artifacts from.  This must exactly match the job name on the Jenkins server."
+      },
+      {
+        "name": "saveTo",
+        "type": "filePath",
+        "label": "Save to",
+        "defaultValue": "jenkinArtifacts",
+        "required": true,
+        "helpMarkDown": "Jenkins artifacts will be downloaded and saved to this directory.  This directory will be created if it does not exist."
+      },
+      {  
+        "name": "trustSSL",  
```

Comment:
Yes, we plan to replace this input with a "Trust Server Certificate" setting on the Jenkins Service Endpoint.  This is a stop-gap.  Some people need this capability immediately and we couldn't squeeze the Service Endpoint changes into this sprint and perhaps not even next sprint.